sbc_compassion/models/sponsorship_correspondence.py

```diff
@@ -259,6 +265,15 @@ def _set_partner_review(self):
             if letter.correspondant_id.mandatory_review:
                 letter.mandatory_review = True
 
+    @api.depends('page_ids')
+    def _compute_page(self):
+        self.original_text = ''
```

Comment:
Put original_text and translated_text into a variable and assign `self.original_text = original_text` only at the end of the method. This will be more efficient.